packages/prbnmcn-dagger/prbnmcn-dagger.0.0.4/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Probabilistic programming library"
+description: """
+A library for probabilistic programming. It takes the form of a signature for a monadic language exposing sampling and scoring primitives together with inference algorithms implementing that signature, including:
+- sequential Monte Carlo (SMC)
+- single-site lightweight Metropolis-Hastings"""
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "MIT"
+tags: ["statistics"]
+homepage: "http://github.com/igarnier/prbnmcn-dagger"
+bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "5.0.0"}
+  "domainslib" {>= "0.5"}
+  "prbnmcn-cgrph" {= "0.0.2"}
+  "odoc" {with-doc}
+]
```

Comment:
```suggestion
]
conflicts: ["ocaml-option-bytecode-only"]
```

This will restrict the package to 64-bit architectures only by only allowing it to compile on native code architecturs, since there are no native code 32-bit architectures on OCaml 5.0.0+

Reply:
Good to know, thanks!